asc.appdata.xml.in

```diff
@@ -10,12 +13,22 @@
             ASC can be played both against the AI and against other human players, using hotseat or PlayByMail. It is also used to run the multiplayer universe Project Battle Planets.
         </p>
     </description>
+    <content_rating type="oars-1.1">
+        <content_attribute id="violence-fantasy">moderate</content_attribute>
```

Comment:
I'm surprised seeing ASC getting the 2nd highest rating regarding violence.

Checking the definition of OARS, I agree that this is aggressive conflict. I could argue that ASC is, just like chess, abstracted to a degree where we don't really have _characters_  - but as this is my first encounter with any content rating system, I'll just accept your proposal for the time being, assuming that you know what you are doing :-)

Reply:
Yes, this is debatable and also a reason why this should be maintained upstream. I mostly eyeballed it. I thought war would count as aggressive conflict. Also wasn't sure about in-game text messaging because I didn't find the function. If it exists, this also should be noted.